test/e2e/app-dir/dynamic-io/dynamic-io.test.ts

```diff
@@ -66,15 +66,15 @@ describe('dynamic-io', () => {
       expect(await hasStaticIndicator(browser)).toBe(true)
     })
 
-    it('should have static indicator on not-found route', async () => {
+    it('should not have static indicator on not-found route', async () => {
```

Comment:
why did this change?

Reply:
streaming meatdata + dynamicIO behaves differently.
reverted and disabled streaming metadata for dynamicIO